nautobot/docs/user-guides/getting-started/index.md

```diff
@@ -5,14 +5,14 @@ This guide will demonstrate how to use and perform common operations in Nautobot
 
 It is intended to assist the user with:
 
-* Understanding the prerequisites to adding a new Device
+* Understanding the prerequisites for dding a new Device
```

Comment:
```suggestion
* Understanding the prerequisites for adding a new Device
```

Reply:
'prerequisites for adding a device' or 'prerequisites to add a device' are correct (I think); I don't think 'prerequisites to adding' is right.

fixed the 'adding' misspelling